applications/asset_tracker_v2/src/modules/util_module.c

```diff
@@ -229,17 +229,18 @@ static void reboot_ack_check(uint32_t module_id)
 /* Message handler for STATE_INIT. */
 static void on_state_init(struct util_msg_data *msg)
 {
-	if (IS_EVENT(msg, cloud,  CLOUD_EVT_FOTA_DONE)) {
+	if ((IS_EVENT(msg, cloud, CLOUD_EVT_FOTA_DONE))) {
```

Comment:
Is the extra parenthesis needed?

Reply:
Not anymore, they're relics from testing out a different approach